### PR TITLE
Dreamview: updated usage stable

### PR DESCRIPTION
**Pull request by vlin17**

### CODE REVIEW

**Comment by siyangy** [comment]
@natashadsouza can you put your comment on the original changes? You can do this by switching to 'Files changed' tab.

File: docs/specs/dreamview_usage_table.md [hunk 1]
```diff
@@ -10,9 +10,11 @@ Dreamview is a web application that,
 The application layout is divided into several regions: header, sidebar, main view, and tool view.
 
 ### Header
-The Header has 4 drop-downs that can be set as shown:
+The Header has 3 drop-downs that can be set as shown:
 ![](images/dreamview_usage_table/header.png) 
 
+The Co-Driver switch is used to detect disengagement event automatically. Once detect, dreamview will pop up the data recorder window for co-driver to enter a new drive event.
```
Comment (natashadsouza) [inline docs/specs/dreamview_usage_table.md]: The Co-Driver switch is used to the detect disengagement event automatically. Once detected, Dreamview will display a pop-up of the data recorder window for the co-driver to enter a new drive event.

**Comment by natashadsouza** [comment]
@siyangy @vlin17 comment added to the document directly.

File: docs/specs/dreamview_usage_table.md (hunk 1 above)
Comment (vlin17) [reply docs/specs/dreamview_usage_table.md]: Changed. Thanks for the correction.

**Pull request merged**